frontend/src/pages/AboutPage.js

```diff
@@ -435,4 +460,6 @@ export {
   TextH1,
   LI,
   howToSaveStagedChanges /*HelpWidget, HelpButton,*/,
+  axiosPost,
```

Comment:
Hmm, I'm not sure these two exports are needed. `axiosPost` should come from importing the `axios` library here and I would imagine doesn't need to be exported. `handleClick` I don't think should be needed anywhere else either.

Actually, now that I think of it, `handleClick` is too generic of a name. Can you change it to `handleRefresh`?